src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.tasks.Task;
+
+
+
+/**
+ * Storage class deals with loading tasks from the file and saving tasks in the file.
+ */
+public class Storage {
+    private final String filePath;
+
+    /**
+     * Constructor for Storage.
+     *
+     * @param filePath The filepath where data is saved.
+     */
```

Comment:
I noticed you did a good job at ensuring every fullstops at the end of each statement for javadocs which might have been commonly overlooked 👍 👍